riscv-c-api.md

```diff
@@ -705,6 +705,89 @@ statements, including both RISC-V specific and common operand modifiers.
 
 ## Function Multi-version
 
-Function multi-versioning(FMV) provides an approach to selecting the appropriate function according to the runtime environment. The final binary may contain all versions of the function, with the compiler generating all supported versions and the runtime selecting the appropriate one.
+Function multi-versioning (FMV) allows selecting the appropriate function based on the runtime environment. The binary may contain multiple versions of the function, with the compiler generating all supported versions and selecting the appropriate one at runtime.
 
-This feature is triggered by `target_version/target_clones` function attribute.
+This feature is activated by the `target_version/target_clones` function attributes.
+
+### Extension Bitmask
+
+When generating the resolver function for FMV, a method is required to retrieve environment information. This is achieved through two bitmask structures: `__riscv_feature_bits` for standard extensions and `__riscv_vendor_feature_bits` for vendor-specific extensions.
+
+The bitmask structures use the following definitions:
+
+```c
+struct {
+    unsigned length;
+    unsigned long long features[];
+} __riscv_feature_bits;
+
+struct {
+    unsigned vendorID;
+    unsigned length;
+    unsigned long long features[];
+} __riscv_vendor_feature_bits;
+```
+
+- `length`: Represents the number of elements in the features array.
+- `features`: An `unsigned long long` array where each bit indicates 1 for a specific extension enabled by the system or 0 for the extension's status unknown in system.
+- `vendorID`: Indicates the current vendor core.
+
+To initiate these structures based on the system's extension status, the following function is provided:
+
+```
+void __init_riscv_features_bits();
```

Comment:
Now we have an `s` after `feature` and `bit` while the variable `__riscv_feature_bits` only has it after `bit`